src/Plugin/EntityMetaRelation/ListPage.php

```diff
@@ -230,12 +230,25 @@ public function build(array $form, FormStateInterface $form_state, ContentEntity
     // Get available filters.
     if ($list_source && $available_filters = $list_source->getAvailableFilters()) {
       $configuration = $this->getExposedFiltersConfiguration($list_source, $entity_meta_wrapper);
+
+      // Override checkbox.
+      $form[$key]['bundle_wrapper']['exposed_filters_wrapper']['exposed_filters_override'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Override default exposed filters'),
```

Comment:
`Override default exposed filters`

Let's add also a description: `Configure which exposed filters should show up on this page.`